### PR TITLE
Blog: add Hennadii post

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

**Comment by harding** [comment]
Looks great!  Thanks for all your work, @hebasto!

File: _posts/2021-04-22-hennadii.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+_We talked to Brink grant recipient Hennadii Stepanov about how he started
+contributing to Bitcoin Core and his priorities for the next year._
+
+I first heard about Bitcoin from friends at the beginning of 2017. I had some
+Linux system admin experience and at the time was maintaining a Java-based GUI
+of an accounting application for a credit union. Eventually, I spun up an old
+laptop to run Bitcoin Core but the laptop happened to have a small HDD and space
+was limited. I discovered that Bitcoin Core had a pruning option, which I tried
+using. While reviewing the related code, I spotted a bug in the documentation
+and decided to file my first Bitcoin Core pull request, [#18344][bitcoin core 18344]. It
```
Comment (hebasto) [inline _posts/2021-04-22-hennadii.md:20]: s/18344/13844/g

File: _posts/2021-04-22-hennadii.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Lastly, I will continue to keep the [Ukrainian language translation][transifex
+bitcoin] for the Bitcoin Core project up to date since it was previously
+abandoned.
+
+I’m proud of the contributions I made while [sponsored by CardCoins and
+Payvant][cardcoins hebasto blog] and look forward to continuing my work with
+Brink’s support.
+
+[bitcoin core 18344]: https://github.com/bitcoin/bitcoin/pull/13844
```
Comment (hebasto) [inline _posts/2021-04-22-hennadii.md:60]: s/18344/13844/

**Comment by hebasto** [comment]
Many thanks to all Brink team for your support!

**Comment by jnewbery** [comment]
Made minor edits to title and excerpt, and also changed the capitalization on a different blog post's title since I noticed it was inconsistent.

**Review by jnewbery — APPROVED** [review]

**Comment by bitschmidty** [comment]
ACK ebcef7bdd59b51ba50062ebb38780042f763f64d will publish tomorrow.

**Comment by bitschmidty** [comment]
Squashed.

**Pull request merged**